aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -647,6 +663,24 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 			},
 		},
 
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
Is there a particular reason to implement the customizable timeouts here? In general, we prefer not add them unless there is a scaling dimension (e.g. data size) that affects the amount of time an operation takes.

Reply:
Done.